child_compassion/mappings/childpool_search_mapping.py

```diff
@@ -48,3 +49,17 @@ def _process_connect_data(self, connect_data):
         for key, value in temp_data.iteritems():
             if not value:
                 del connect_data[key]
+
+
+class ChilpoolSearchMixMapping(OnrampMapping):
+    """
+    Childpool mapping for searching mix of available children from GMC.
+    """
+    ODOO_MODEL = 'compassion.childpool.search'
+    ACTION = "rich_mix"
+
+    CONNECT_MAPPING = {
+        'numberOfBeneficiaries': 'take',
+    }
+
+    FIELDS_TO_SUBMIT = {k: None for k in CONNECT_MAPPING.iterkeys()}
```

Comment:
since you have only one field, is it more simple to put it ? `'numberOfBeneficiaries': None`